ui_framework/src/components/dial_menu/_index.scss

```diff
@@ -0,0 +1,5 @@
+// Dial menu is a phone like dial comprised of an svg icon and a title.
+
+$kuiDialMenuWidth: $kuiSize * 6;
```

Comment:
I think I got it! The word we're looking for is [keypad](https://www.google.com/search?q=keypad&safe=active&rlz=1C5CHFA_enUS695US695&source=lnms&tbm=isch&sa=X&ved=0ahUKEwi_xrT9lpvVAhWBcCYKHfAbCDgQ_AUICigB&biw=2560&bih=1315). How about `KeyPadMenu`? I think something less tied to a phone-interface would be ideal (e.g. `GridMenu`), but I can compromise. 😄